dev/ci/user-overlays/17304-LasseBlaauwbroek-17295-attempt-two.sh

```diff
@@ -0,0 +1 @@
+overlay coq_library_undecidability https://github.com/LasseBlaauwbroek/coq-library-undecidability overlay-17304 17304
```

Comment:
If you add your branch name at the end it will get auto detected locally
```suggestion
overlay coq_library_undecidability https://github.com/LasseBlaauwbroek/coq-library-undecidability overlay-17304 17304 fix-17295-attempt-two 
```
note that usually the overlay branch and the coq branch use the same name, when that's the case there's no need for the last argument and it autodetects based on the overlay branch.

Reply:
Thanks! (I'll not commit this now, to avoid running the CI again.